easylinks/src/main/java/com/google/step/servlets/EasyLinkServlet.java

```diff
@@ -0,0 +1,39 @@
+package com.google.step.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responses to redirect the easy link to another website*/
+@WebServlet("/easy-link")
+public class EasyLinkServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String[] easyLink = ServletHelper.getParameterWithDefault(request, "easy-link", "").split("/");
+    // Process the easy link based on its type
+    switch (easyLink[0])
```

Comment:
If easyLink is empty, we will have index out of bound here

Reply:
Done! If the link is empty, the first element of the array will be an empty string, we could check that and redirect to home page.